src/integTest/resources/control_cycloneDx.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+  id 'java'
+  id 'org.sonatype.gradle.plugins.scan'
+}
+
+repositories {
+  mavenCentral()
+}
+dependencies {
+  implementation 'commons-collections:commons-collections:3.1'
+}
+
+nexusIQScan {
+  serverUrl = 'http://localhost:8081'
+  username = 'admin'
+  password = 'admin123'
+  applicationId = 'testing-gradle-plugin'
+  simulationEnabled = true
+}
+
+ossIndexAudit {
+  simulationEnabled = true
+  outputFormat = "JSON_CYCLONE_DX_14"
```

Comment:
nitpicking but probably should be single quotes.

Reply:
Good catch!
480dca7e738e29324831d485fad25123dbb91741